openvino_tensorflow/rewrite_pass.cc

```diff
@@ -158,6 +158,13 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
         disabled_ops_set.insert(batched_disabled_ops[i]);
       }
     }
+
+    //disable TopKV2 as of now as it impacts performance for TF_HUB object detection models
+    disabled_ops_set.insert("TopKV2");
+    for (auto itr = disabled_ops_set.begin(); itr!=disabled_ops_set.end(); itr++) {
```

Comment:
I think this for loop can be removed for now and a single log for "TopKV2" can be added, and which could print a message like "Disabled OP - TopKV2".